src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/Runtime/DetectPreviewFeatureUnitTests.cs

```diff
@@ -490,352 +551,344 @@ static void Main(string[] args)
+            test.ExpectedDiagnostics.Add(VerifyCS.Diagnostic(DetectPreviewFeatureAnalyzer.GeneralPreviewFeatureAttributeRule).WithLocation(0).WithArguments("Del"));
+            await test.RunAsync();
 
         }
 
         [Fact]
         public async Task TestEnumValue()
         {
             var csInput = @" 
-using System.Runtime.Versioning; using System;
-namespace Preview_Feature_Scratch
-{" +
+        using System.Runtime.Versioning; using System;
+        namespace Preview_Feature_Scratch
+        {" +
 @"
 
-    enum AnEnum
-    {
-        [RequiresPreviewFeatures]
-        Foo,
-        Bar
-    }
+            enum AnEnum
+            {
+                [RequiresPreviewFeatures]
+                Foo,
+                Bar
+            }
 
-    class Program
-    {
-        public Program()
-        {
-        }
+            class Program
+            {
+                public Program()
+                {
+                }
 
-        static void Main(string[] args)
-        {
-            AnEnum fooEnum = [|AnEnum.Foo|];
-        }
-    }
-}";
+                static void Main(string[] args)
+                {
+                    AnEnum fooEnum = {|#0:AnEnum.Foo|};
+                }
+            }
+        }";
 
-            await TestCS(csInput);
+            var test = TestCS(csInput);
+            test.ExpectedDiagnostics.Add(VerifyCS.Diagnostic(DetectPreviewFeatureAnalyzer.GeneralPreviewFeatureAttributeRule).WithLocation(0).WithArguments("Foo"));
+            await test.RunAsync();
         }
 
         [Fact]
         public async Task TestEnumValue_NoDiagnostic()
         {
             var csInput = @" 
-using System.Runtime.Versioning; using System;
-namespace Preview_Feature_Scratch
-{" +
+        using System.Runtime.Versioning; using System;
+        namespace Preview_Feature_Scratch
+        {" +
 @"
 
-    enum AnEnum
-    {
-        Foo,
-        [RequiresPreviewFeatures]
-        Bar
-    }
+            enum AnEnum
+            {
+                Foo,
+                [RequiresPreviewFeatures]
+                Bar
+            }
 
-    class Program
-    {
-        public Program()
-        {
-        }
+            class Program
+            {
+                public Program()
+                {
+                }
 
-        static void Main(string[] args)
-        {
-            AnEnum fooEnum = AnEnum.Foo;
-        }
-    }
-}";
+                static void Main(string[] args)
+                {
+                    AnEnum fooEnum = AnEnum.Foo;
+                }
+            }
+        }";
 
-            await TestCS(csInput);
+            var test = TestCS(csInput);
+            await test.RunAsync();
         }
 
         [Fact]
         public async Task TestEnum()
         {
             var csInput = @" 
-using System.Runtime.Versioning; using System;
-namespace Preview_Feature_Scratch
-{" +
+        using System.Runtime.Versioning; using System;
+        namespace Preview_Feature_Scratch
+        {" +
 @"
 
```

Comment:
NIT: why all tests have this gap? Seems all could be removed
```suggestion
        {
```